Documentation/capabilities-guide.md

```diff
@@ -222,6 +223,101 @@ root
 nc: bind: Permission denied
 ```
 
+## Overriding capabilities
+
+Capabilities sets are typically defined when creating images, as they are tightly
+linked to specific app requirements. However, image consumers may need to further
+tweak/restrict the set of available capabilities in specific local scenarios.
+This can be done either by permanently patching the manifest of specific images, or
+by overriding capability isolators via command line arguments.
+
+### Patching images
+
+Image manifests can be manipulated manually, by unpacking the image and editing
+the manifest file, or via helper tools like `actool`.
+In order to override pre-defined capabilities set, it is enough to replace any
+existing capability isolators with a new custom one.
+
+`actool patch-manifest` provides two options to manipulate the retain set
+(`-capability`) and the remove set (`-revoke-capability`).
+Starting from the `caps-retain-set-example.aci` image above, it is possible to
+transform its capabilities set as follows:
+
+```
+$ actool cat-manifest caps-retain-set-example.aci
+...
+    "isolators": [
+      {
+        "name": "os/linux/capabilities-retain-set",
+        "value": {
+          "set": [
+            "CAP_NET_RAW"
+          ]
+        }
+      }
+    ]
+...
+
+$ actool patch-manifest -capability CAP_NET_RAW,CAP_NET_BIND_SERVICE caps-retain-set-example.aci caps-retain-set-patched.aci
+
+$ actool cat-manifest caps-retain-set-patched.aci
+...
+    "isolators": [
+      {
+        "name": "os/linux/capabilities-retain-set",
+        "value": {
+          "set": [
+            "CAP_NET_RAW",
+            "CAP_NET_BIND_SERVICE"
+          ]
+        }
+      }
+    ]
+...
+
+```
+
+And then verify that the additional `CAP_NET_BIND_SERVICE` capability is now
```

Comment:
Now run the image to check that the `CAP_NET_BIND_SERVICE` capability added to the patched image is retained as expected: